Dynamic-Programming/UniquePaths.js

```diff
@@ -0,0 +1,41 @@
+
+/*
+ *
+ *  Unique Paths
+ *
+ *  There is a robot on an `m x n` grid.
+ *  The robot is initially located at the top-left corner.
+ *  The robot tries to move to the bottom-right corner.
+ *  The robot can only move either down or right at any point in time.
+ *
+ *  Given the two integers `m` and `n`,
+ *  return the number of possible unique paths that the robot can take to reach the bottom-right corner.
+ *  More info: https://leetcode.com/problems/unique-paths/
+ */
+
+/*
+ * @param {number} m
+ * @param {number} n
+ * @return {number}
+ */
+
+const uniquePaths = (m, n) => {
+  // only one way to reach end
+  if (m === 1 || n === 1) return 1
+
+  // build a mxn grid
```

Comment:
Note that you don't even need the full grid; keeping just the last row & the last value suffices.

Reply:
Thanks for this suggestion